decode-entity.browser.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-env browser */
+var el
+
+module.exports = decodeEntity
+
+function decodeEntity(characters) {
+  el = el || document.createElement('em')
```

Comment:
Could you write the `var` declarations at the top, and use them later?  Such as:

```js
var entity = '&' + characters + ';'
var char

el = el || document.createElement('em')
el.innerHTML = entity
char = el.textContent
```